xave-finance/halodao-admin-scripts
# hdev-703-xav-airdrop

## Description of Changes



Added script to generate addresses eligible for XAV airdrop
[Link to Jira Ticket](https://halodao.atlassian.net/browse/HDEV-703)

## How To Test
Run the script
yarn hardhat xav-airdrop --network mainnet

Can use etherscan's  token check tool to verify address balance at block height.
https://etherscan.io/tokencheck-tool

### Developer Checklist:


### Reviewers Checklist:
